doc/man3/EVP_PKEY_is_a.pod

```diff
@@ -0,0 +1,74 @@
+=pod
+
+=head1 NAME
+
+EVP_PKEY_is_a, EVP_PKEY_can_sign
+- key type and capabilities functions
+
+=head1 SYNOPSIS
+
+ #include <openssl/evp.h>
+
+ int EVP_PKEY_is_a(const EVP_PKEY *pkey, const char *name);
+ int EVP_PKEY_can_sign(const EVP_PKEY *pkey);
+
+=head1 DESCRIPTION
+
+EVP_PKEY_is_a() checks if the key type of I<pkey> is I<name>.
+
+EVP_PKEY_can_sign() checks if I<pkey> can be used for signing.  The
```

Comment:
I assume this returns false if the key can only be used for verifying? e.g. because it is a public key? Could we make that more explicit?

Reply:
Ah, so having it verify that `evp_keymgmt_has(p->keymgmt, p->keydata, OSSL_KEYMGMT_SELECT_PRIVATE_KEY)` is true?  Yeahsure

Reply:
See my comment above...